source/support.html.erb

```diff
@@ -90,18 +95,17 @@ title: Support
       <h2 class="heading-large">Contacting us about a critical issue</h2>
 
       <p>
-        To contact us about a critical issue during working hours, please use
-        our support email address,
+        To contact us during working hours, use our support email address,
         <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">
           gov-uk-paas-support@digital.cabinet-office.gov.uk
         </a>.
       </p>
 
       <p>
-        To contact us outside of working hours, please use the emergency email
-        address you have been given. This will raise a high priority alert and
-        could wake us up in the middle of the night, so it should only be used
-        for critical incidents.
+        To contact us about a critical issue outside of working hours, please
+        use the emergency email address you have been given. This will raise a
+        high priority alert and could wake us up in the middle of the night, so
```

Comment:
Instead of saying "...and could wake us up in the middle of the night", maybe say "This is monitored 24 hours a day and will raise a high priority alert, so it should only be used for critical incidents." Just to de-personalise it slightly, unless that's the preferred style.